### PR TITLE
fix margins & rename Loading component to Spinner

### PR DESCRIPTION
**Pull request by MikeAdamczyk**

### CODE REVIEW

File: src/common/SinglePage/index.js [hunk 1]
```diff
@@ -32,10 +32,11 @@ import spinner from "../../images/icon-spinner.svg";
 import { getProductionYear } from "../../lib/utils";
 import { QUERY_PARAMETER } from "../../lib/consts";
 import { DetailsContainer } from "./styled";
+import { ListContainer, Wrapper } from "../Containers/styled";
+import { Spinner, SpinnerBox } from "../Spinner/styled";
 import { Wrapper, Slider } from "../Containers/styled";
```
Comment (anetaszynal) [inline src/common/SinglePage/index.js:37]: Why are here two the same imports?

File: src/common/SinglePage/index.js [hunk 2]
```diff
@@ -192,6 +196,7 @@ export const SinglePage = ({match, detailType, listType}) => {
                         </Slider>                       
                     }
                 </Wrapper>
+
```
Comment (anetaszynal) [inline src/common/SinglePage/index.js:199]: I think that this empty lines are unnecessary ;)

**Review by anetaszynal — APPROVED** [review]
It looks good ;)

File: src/common/SinglePage/index.js (hunk 1 above)
Comment (MikeAdamczyk) [reply src/common/SinglePage/index.js:36]: amended

File: src/common/SinglePage/index.js (hunk 2 above)
Comment (MikeAdamczyk) [reply src/common/SinglePage/index.js:199]: it's for better visibility, the solid block of code is really unreadable ;)

**Pull request merged**